gradoop-flink/src/main/java/org/gradoop/flink/model/api/epgm/LogicalGraphOperators.java

```diff
@@ -662,7 +662,7 @@ GraphCollection groupEdgesByRollUp(
    * @param operator unary graph to graph operator
    * @return result of given operator
    */
-  LogicalGraph callForGraph(UnaryGraphToGraphOperator operator);
+  LogicalGraph callForGraph(UnaryBaseGraphToGraphOperator<LogicalGraph> operator);
```

Comment:
Can be replaced by UnaryGraphToGraphOperator, which is identical to UnaryBaseGraphToGraphOperator<  LogicalGraph >.